app/views/handlers/list_action.py

```diff
@@ -15,9 +15,15 @@ def parent_location(self):
             block_id=self.rendered_block["parent_id"],
         )
 
+    @property
+    def section_routing_path(self):
+        return self.router.section_routing_path(
+            section_id=self._current_location.section_id
+        )
+
     def is_location_valid(self):
         can_access_parent_location = self.router.can_access_location(
-            self.parent_location, self._routing_path
+            self.parent_location, self.section_routing_path
         )
```

Comment:
I don't get why this has changed?. `section_routing_path` shouldn't be property of `ListAction`. `self._routing_path` is available due to inheritance, so we shouldn't calculate it again.